views/BoggleView.java

```diff
@@ -373,14 +388,187 @@ private GridPane populateGridPane(int rSize, String userString ){
                     wordInput.setText(inputWord[0]);
                     System.out.println(inputWord[0]);
                 });
+                mainButtons.add(newButton);
                 grid.add(newButton, row, col);
             }
         }
 
+        makeSpecialButtons();
         return grid;
+    }
+
+
+
+    /*
+        Creates special buttons on the grid view for accessibility purposes.
+     */
+
+    private void makeSpecialButtons() {
+
+        // increase font button
+        Button fontUp = new Button("Increase Font");
+        fontUp.setId("Increase Font");
+        fontUp.setPrefSize(100,50);
+        fontUp.setFont(new Font(this.font));
+        fontUp.setStyle("-fx-background-color: #10871b; -fx-text-fill: white;");
+
+        borderPane.setLeft(fontUp);
```

Comment:
This was great, but it might cause a conflict. Maybe add most of your changes to the bottom